### DISCUSSION TITLE
Supported Model in README

### DISCUSSION THREAD

**Discussion by sanjeev-bhandari** [mobiusml/hqq#60] [Q&A] [answered]
Can you please update the README, specifying which model do you support? All LLMs from the hugging face Or specific architecture?
If not will you support other model in future?
![image](https://github.com/mobiusml/hqq/assets/160564883/06a2beef-2c08-4be3-a30b-277735b087ac)

Extended from this issue: https://github.com/mobiusml/hqq/issues/30

By the way, this is great library for quantizing model.

**Comment by mobicham** [comment, marked as answer]
Hi, I think it's quite clear. 
If a Hugging Face model is not on the list (llama, Mistral, Mixtral, Phi), use auto mode: https://github.com/mobiusml/hqq/?tab=readme-ov-file#auto-mode-1
Auto-mode basically tries to figure out how to quantize stuff automatically, but it's not guaranteed to work.
